js/webui.js

```diff
@@ -2125,46 +2127,69 @@ var theWebUI =
 		}
 	},
 
+	resetLabels: function() {
+		var allLbls = $('.-_-_-all-_-_-');
+		for(var i = 0; i < allLbls.length; i++)
+			this.switchLabel(allLbls.get(i));
+	},
+
 	switchLabel: function(obj)
 	{
-		if(obj.id != this.actLbl)
+		var panelCont = obj.closest(".catpanel_cont");
```

Comment:
AFAIK, method 'closest' is not supported by IE11. Can you replace it with a jQuery analog?